recipes/autowrap/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "autowrap" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "4ba2b9486434eb23306dfbceb62fb82eca9e17f5a0c69babf0c5ad97cfdaef7c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython ==0.21
+  run:
+    - python
+    - setuptools
+    - cython ==0.21
+
+test:
+  imports:
+    - autowrap
+    - autowrap.data_files
+
+  commands:
+    - autowrap --version
+
+about:
+  home: https://github.com/uweschmitt/autowrap
+  license: BSD-3-Clause
+  license_family: BSD
+  # autowrap does not include its license file in the pypi release.
```

Comment:
Might be worth noting that its license file now included in the MANIFEST.in and so